taxonomies/additives_classes.txt

```diff
@@ -6,18 +6,19 @@ stopwords:en:a,an,for,added,to
 stopwords:fr:la,les,des,du,de,d,un,une,ajouté,ajoutés,ajoutée,ajoutées,pour
 stopwords:es:de,del,la,las,los,una
 
-en: Acid, Acidifier
+en: Acid, Acidifier, corrector of acidity
```

Comment:
there is a special separate class for corrector of acidity: en: Acidity regulator, alkali, base, buffer, buffering agent, pH adjusting agent